sdl2promela/stop_condition/translator.py

```diff
@@ -447,6 +474,62 @@ def _find_first_variable(
     """
     index = 0
 
+    if isinstance(selector.elements[0], model.CallExpression):
+        call_expression: model.CallExpression = selector.elements[0]
+        if len(call_expression.parameters) == 1 and isinstance(
+            call_expression.parameters[0], model.Selector
+        ):
+            nested_selector: model.Selector = call_expression.parameters[0]
+            if len(nested_selector.elements) == 2:
```

Comment:
Do we have else for this?

Reply:
Maybe just throw an exception at the end, if the function did not return properly

Reply:
refactored